### PR TITLE
Fix linter issues

### PR DESCRIPTION
**Pull request by palfrey**

### CODE REVIEW

**Comment by palfrey** [comment]
@epwalsh This should fix the current lint failures on a bunch of the PRs

File: dev-requirements.txt [hunk 1]
```diff
@@ -5,4 +5,4 @@ flake8
 mypy==0.910
 
 # Automatic code formatting
-black==21.9b0
+black==22.3.0
```
Comment (palfrey) [inline dev-requirements.txt:8]: Black 21.9b0 breaks now (well, without fixing the click version).

**Comment by Miezhiko** [comment]
line 410 `let name = &pat.ident.to_string();` also look very strange to me, why not `pat.ident`

**Comment by palfrey** [comment]
> line 410 `let name = &pat.ident.to_string();` also look very strange to me, why not `pat.ident`

No idea, but doesn't appear to be one that clippy complains about so a problem for another PR!

**Comment by palfrey** [comment]
Any thoughts @epwalsh?

**Comment by epwalsh** [comment]
Hey @palfrey, sorry, I don't have the bandwidth to help maintain this project at the moment ☹️

**Comment by palfrey** [comment]
Obsoleted by https://github.com/rusty-celery/rusty-celery/pull/324

**Pull request closed without merging**